dbms/SQLscriptFile.sql

```diff
@@ -0,0 +1,75 @@
+CREATE DATABASE ecommerceDATABASE;
+USE ecommerceDATABASE;
+
+CREATE TABLE Customers (
+    CustomerID INT PRIMARY KEY AUTO_INCREMENT,
+    FirstName VARCHAR(50),
+    MiddleName VARCHAR(50),
+    LastName VARCHAR(50)
+);
+
+ CREATE TABLE Contact (
+        ContactID INT PRIMARY KEY,
+        CustomerID INT,
+        Email VARCHAR(100),
+        Phone VARCHAR(20),
```

Comment:
Instead of having individual columns for `Email` and `Phone`, you can use the `Type` and `Info` columns.

This modeling allows you to store other kinds of contact information as well for example fax number